### PR TITLE
notification before event

### PR DESCRIPTION
**Pull request by mv-v-7**

### CODE REVIEW

**Review by aLLeXUs — COMMENTED** [review]
Please apply discussed changes

**Comment by mv-v-7** [comment]
Please notice for correct work of feature correct QUEUE config needed and Starting The Scheduler

File: api/app/Jobs/SendNotificationBeforeEvent.php [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+            new NotifiedCriterion(false)
+           ];
+        $events = $eventRepository->findByCriteria(...$criteria);
+
+        foreach ($events as $event) {
+            $event->eventType->owner->notify(new NotificationBeforeEventForOwner($event));
+            Notification::route('mail', $event->invitee_email)
+                ->notify(new NotificationBeforeEventForInvitee($event));
+            $event->notified = true;
+            $event->save();
```
Comment (lenchvolodymyr) [inline api/app/Jobs/SendNotificationBeforeEvent.php:46]: saving via repository

File: api/app/Jobs/SendNotificationBeforeEvent.php [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+    public function __construct()
+    {
+    }
+
+    public function handle(EventRepository $eventRepository)
+    {
+        $now = new Carbon();
+        $tenMinutesLater = $now->addMinutes(10);
+        $criteria = [
```
Comment (lenchvolodymyr) [inline api/app/Jobs/SendNotificationBeforeEvent.php:35]: why EndDate? We need notify user before the event starts

**Review by lenchvolodymyr — CHANGES_REQUESTED** [review]

File: api/app/Jobs/SendNotificationBeforeEvent.php (hunk 2 above)
Comment (mv-v-7) [reply api/app/Jobs/SendNotificationBeforeEvent.php:34]: Created new criteria with more suitable names.

**Review by lenchvolodymyr — APPROVED** [review]

**Pull request merged**